docs/root/intro/arch_overview/other_protocols/other_protocols.rst

```diff
@@ -9,3 +9,4 @@ Other protocols
   dynamo
   redis
   postgres
+  smtp
```

Comment:
I think this should be linked in `docs/root/api-v3/config/contrib/contrib.rst` not here, otherwise it's not clear which extensions are `contrib`. See docs/root/api-v3/config/contrib/contrib.rst.

Reply:
done